lib/puppet/type/node_encrypted_file.rb

```diff
@@ -1,4 +1,4 @@
-require 'puppet_x/binford2k/node_encrypt'
+require File.join(File.dirname(__FILE__), '../..', 'puppet_x/binford2k/node_encrypt.rb')
```

Comment:
Did you validate this on a windows node? It *should* work, since the win32 filesystem APIs all recognize the forward slash as a directory separator. But I'd like to validate that before merging.

Reply:
I sadly don't have access to a Windows node as of yet... 
We are still fighting with the Puppet 5 upgrade and improving Foreman integration here ;-).

Reply:
I think adding a 

`require File.join(File.dirname(File.dirname(File.dirname(__FILE__))), 'puppet_x/binford2k/node_encrypt.rb')` would help on windows.